device/bgpadvanced.yaml

```diff
@@ -0,0 +1,46 @@
+components:
+  schemas:
+    Device.BgpAdvanced:
+      x-status: under-review
+      description: >-
+        Container for BGP advanced settings.
+      type: object
+      properties:
+        hold_time_interval:
+          description: >-
+            Number of seconds the sender proposes for the value of the Hold Timer
+          type: integer
+          default: 90
+        keep_alive_interval:
+          description: >-
+            Number of seconds between transmissions of Keep Alive messages by router
+          type: integer
+          default: 30
+        update_interval:
+          description: >-
+            TBD
+          type: integer
+          default: 0
```

Comment:
Q: 0 is the correct default here?  Does it signify something like send updates ASAP?

Reply:
@apratimmukherjee ?

Reply:
Yes, 0 is the correct default since we want  to normally send updates as fast as possible unless the DUT is not able to handle it.